openedx/core/djangoapps/credentials/tests/test_utils.py

```diff
@@ -35,41 +39,23 @@ def setUp(self):
         ClientFactory(name=CredentialsApiConfig.OAUTH2_CLIENT_NAME, client_type=CONFIDENTIAL)
         ClientFactory(name=ProgramsApiConfig.OAUTH2_CLIENT_NAME, client_type=CONFIDENTIAL)
         self.user = UserFactory()
+        self.primary_uuid = str(uuid.uuid4())
+        self.alternate_uuid = str(uuid.uuid4())
 
         cache.clear()
 
-    def _expected_progam_credentials_data(self):
-        """
-        Dry method for getting expected program credentials response data.
-        """
-        return [
-            factories.UserCredential(
-                id=1,
-                username='test',
-                credential=factories.ProgramCredential(),
-                certificate_url=self.CREDENTIALS_API_RESPONSE['results'][0]['certificate_url'],
-            ),
-            factories.UserCredential(
-                id=2,
-                username='test',
-                credential=factories.ProgramCredential(),
-                certificate_url=self.CREDENTIALS_API_RESPONSE['results'][1]['certificate_url'],
-            )
-        ]
-
-    def expected_credentials_display_data(self):
+    def expected_credentials_display_data(self, programs):
         """ Returns expected credentials data to be represented. """
-        program_credentials_data = self._expected_progam_credentials_data()
         return [
             {
-                'display_name': self.PROGRAMS_API_RESPONSE['results'][0]['name'],
-                'subtitle': self.PROGRAMS_API_RESPONSE['results'][0]['subtitle'],
-                'credential_url':program_credentials_data[0]['certificate_url']
+                'display_name': programs[0]['title'],
+                'subtitle': programs[0]['subtitle'],
+                'credential_url':programs[0]['credential_url']
             },
             {
-                'display_name': self.PROGRAMS_API_RESPONSE['results'][1]['name'],
-                'subtitle':self.PROGRAMS_API_RESPONSE['results'][1]['subtitle'],
-                'credential_url':program_credentials_data[1]['certificate_url']
+                'display_name': programs[1]['title'],
+                'subtitle': programs[1]['subtitle'],
+                'credential_url':programs[1]['credential_url']
```

Comment:
I'm surprised the linter missed this. You need a space between the colon and the value, here and above.